Task3/EEL_task_3/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/lab_task
+    username: root
```

Comment:
Please add username and password as a env variables, for this project you can leave your values as a default for env variables

Reply:
done